cmd/clusterctl/cmd/completion.go

```diff
@@ -90,6 +215,19 @@ var (
 		"bash": runCompletionBash,
 		"zsh":  runCompletionZsh,
 	}
+
+	bashCompletionFlags = map[string]string{
+		"namespace":                 "__clusterctl_kubectl_get_resource_namespace",
+		"kubeconfig-context":        "__clusterctl_kubectl_config_get_contexts",
+		"from-config-map":           "__clusterctl_kubectl_get_resource_configmap",
+		"from-config-map-namespace": "__clusterctl_kubectl_get_resource_namespace",
```

Comment:
I tried to test this with ` clusterctl config cluster --from-config-map <tab>` without any luck.
What I'm doing wrong?